_pages/algorithmia-enterprise/usage-metrics.md

```diff
@@ -1,11 +1,10 @@
 ---
 layout: article
-title:  "Usage Metrics"
```

Comment:
:bulb: I might suggest "Usage Reporting". We don't have a "Basic Usage Reporting", so the addition of "Advanced" doesn't seem necessary?

Reply:
@StephanieKim thoughts?

Reply:
Nope, lets keep it Advanced Usage Reporting. We agreed on it in product meeting - hernan's call.